src/scenes/main/home/components/features-job-section/index.js

```diff
@@ -8,6 +11,24 @@ export default class FeatureJobSection extends Component {
         <div>
           <p className="fj-title">FEATURED JOBS</p>
           <div className="line" />
+          <div className="fj-list">
+            <Row className="fj-row">
```

Comment:
Use map array to create list of components

Reply:
We will change it when we have a test data from the server. I just do it for testing UI

Reply:
Okay, got it! btw, can you deploy to dev so that I can check UI without run code